cli/build/build_test.go

```diff
@@ -62,8 +72,18 @@ func TestFillCtxAppPathIsFile(t *testing.T) {
 	require.NoError(t, os.Chdir(workDir))
 	defer os.Chdir(wd)
 	var buildCtx BuildCtx
+
+	osType, _ := util.GetOs()
+	appDir := ""
+	switch osType {
+	case util.OsMacos:
+		appDir = filepath.Join("/private", workDir, "app1")
```

Comment:
Why do we need these paths manipulations in tests code? Why cannot we just set TMPDIR env var for our tests under Mac?